### PR TITLE
Add km3pipe

### PR DESCRIPTION
**Pull request by tamasgal**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by tamasgal** [comment]
Posting to confirm I am willing to be a maintainer of this package

**Comment by tamasgal** [comment]
@conda-forge/staged-recipes ready for review!

**Comment by tamasgal** [comment]
@conda-forge/staged-recipes ready for review!

**Review by dopplershift — CHANGES_REQUESTED** [review]
One minor tweak and this is ready.

File: recipes/km3pipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    - ligiermirror=km3pipe.utils.ligiermirror:main
+    - qrunprocessor=km3pipe.utils.qrunprocessor:main
+    - qrunqaqc=km3pipe.utils.qrunqaqc:main
+    - daqsample=km3pipe.utils.daqsample:main
+    - tres=km3pipe.utils.tres:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - numpy >=1.12
```
[the remark this answers is not in the document]
Comment (tamasgal) [reply recipes/km3pipe/meta.yaml]: Ah, good catch, thanks! That's a left-over from the old times. I removed it...

**Comment by dopplershift** [comment]
Are you planning on a new release with that change? Otherwise, we can merge with it put back in the recipe here for now.

**Comment by tamasgal** [comment]
I am about to make a release and update this PR in a few minutes!

**Comment by tamasgal** [comment]
Alright, it should be fine now, let's see if the CI agrees...

**Review by dopplershift — APPROVED** [review]

**Pull request merged**